op-alt-da/daclient.go

```diff
@@ -55,6 +73,14 @@ func (c *DAClient) GetInput(ctx context.Context, comm CommitmentData) ([]byte, e
 	if resp.StatusCode == http.StatusNotFound {
 		return nil, ErrNotFound
 	}
+	if resp.StatusCode == http.StatusTeapot {
+		defer resp.Body.Close()
+		var invalidCommitmentReason [250]byte
```

Comment:
What's the significance of 250?

Reply:
Arbitrary number that I guessed should be enough. I will add a comment. Think its always better to have bounds though, even if arbitrary.

Reply:
[9251a34](https://github.com/Layr-Labs/optimism/pull/45/commits/9251a34831a789525f5e4c726ed63e5a81ee19d6)